docs/source/using_the_ve/virtual_ecosystem_in_static_mode.md

```diff
@@ -0,0 +1,440 @@
+like plants or animals while others remain dynamic. At each model step, static
+components reset to their original state (e.g. eaten leaves magically reappear) —
+like a controlled reset — so any change comes only from the active, evolving parts.
+
+This setup enables **targeted experiments** within the full model framework. For
+example, you can test how vegetation responds to changing temperatures without feedback
+from soil moisture changes, or explore animal behaviour in a fixed vegetation landscape.
+
+By choosing which components evolve over time and which stay fixed, Static Mode helps
+reveal **drivers of ecological change** - helping us understand not just
+**what happens**, but **why**.
+
+```{note}
+The Virtual Ecosystem model can, in theory, be run with any combination of one or more
+components held static (except `core`). However, not all combinations are equally
+meaningful. It's worth
+considering your experimental goals carefully and selecting static–dynamic
+configurations that best support the processes you want to isolate or understand.
+```
+
+## Using the static mode
+
+This example demonstrates how to configure and run the Virtual Ecosystem with selected
+components held static (for more technical details,
+[see](./configuration/config.md#static-models)). Specifically, we focus on **hydrology**
+by keeping the
+microclimate, vegetation, animals, and soils fixed. This isolates hydrological
+processes and so we can examine how individual model parameters and process
+representations influence model behaviour.
+
+We run two experiments:
+
+- HydroDefault: 'hydrology-only' simulation with default configuration
+- HydroDry: 'hydrology-only' simulation with reduced initial soil moisture
+
+In practice this means:
+
+- **Microclimate is fixed**: e.g., temperature and relative humidity stay constant
+- **Plants are static**: e.g. water uptake, transpiration, and interception do not vary
+- **Soil microbes, litter, and animals do not feedback** into hydrology
+- **Only hydrology processes evolve over time** in response to precipitation
+
+Finally, we compare their outputs to the default setup (`ve_example`) where all
+components run dynamically.
+
+```{note}
+This guide focuses on how to technically configure and run static mode, not on
+interpreting the experimental results. This enables you to adapt the setup to suit your
+own analysis goals.
+
+The example uses `%%bash` for demonstration on Linux, but it is compatible with all
+operating systems.
+```
+
+### 1: Run `ve_example` as a baseline
+
+Before setting up static experiments, make sure you can run the model successfully.
+If you haven’t yet installed and executed the example, follow the
+[example instructions](./virtual_ecosystem_in_use.md) to familiarise yourself with the
+setup.
+
+```{code-cell} ipython3
+%%bash
+# Remove any existing VE data directory in the /tmp/ directory
+if [ -d /tmp/ve_example ]; then
+  rm -r /tmp/ve_example
+fi
+```
```

Comment:
I think we should hide this cell - it's only there to avoid cockups on developer machines and this is really the only cell with very bash specific commands.